src/base/static/components/input-form/input-form-category-button.js

```diff
@@ -92,7 +93,7 @@ const InputFormCategoryButton = props => {
               : "unset"};
           `}
         >
-          {categoryConfig.label}
+          <T msg={categoryConfig.label} />
```

Comment:
Test usage of the `T` component.

Reply:
If we wanted to use an `i18next` translation string instead of a Google Translate API call, perhaps we could pass the `i18next` translator function exposed by the `translate` HOC as well as the i18next identifier string into the `T` component, and the `T` component would be smart enough to use that if it's present.

So it could look something like:

`<T t={props.t} i18nextIdentifier={categoryConfig.i18nextIdentifier} />`

where `categoryConfig.i18nextIdentifier` would be derived from a field on the backend model.

But we'd only need to add identifiers in cases where flavors demand human translation.